labman/gui/static/js/plateViewer.js

```diff
@@ -450,54 +447,49 @@ PlateViewer.prototype.updateAllRows = function () {
   for (var i = 0; i < this.rows; i ++ ) {
     this.grid.updateRow(i);
   }
-}
+};
+
 
-PlateViewer.prototype.updateDuplicates = function () {
+PlateViewer.prototype.updateUnknownsAndDuplicates = function () {
```

Comment:
Good catch on refactoring this. I believe some comments in this file refer to the `updateDuplicates` and the `updateUnknowns` methods. It would be good to update/remove where appropriate.